README.md

```diff
@@ -186,20 +193,21 @@ The repository contains a sample `.env` file. Rename this from `.env.sample`.
 
 You can now run `npm run start [exporttype]`. See the table with run commands below. The tool will open your export and will convert this. It retrieves the symbols that are supported with YAHOO Finance (e.g. for European stocks like `ASML`, it will retrieve `ASML.AS` by the corresponding ISIN).
 
-| Exporter    | Run command                         |
-| ----------- | ----------------------------------- |
-| Bitvavo     | `run start bitvavo` (or `bv`)       |
-| BUX         | `run start bux`                     |
-| DEGIRO      | `run start degiro`                  |
-| eToro       | `run start etoro`                   |
-| Finpension  | `run start finpension` (or `fp`)    |
-| Freetrade   | `run start freetrade`  (or `ft`)    |
-| IBKR        | `run start ibkr`                    |
-| Rabobank    | `run start rabobank`                |
-| Schwab      | `run start schwab`                  |
-| Swissquote  | `run start swissquote` (or `sq`)    |
-| Trading 212 | `run start trading212` (or `t212`)  |
-| XTB         | `run start xtb`                     |
+| Exporter      | Run command                         |
+| ------------- | ----------------------------------- |
+| Bitvavo       | `run start bitvavo` (or `bv`)       |
+| BUX           | `run start bux`                     |
+| DEGIRO        | `run start degiro`                  |
+| eToro         | `run start etoro`                   |
+| Finpension    | `run start finpension` (or `fp`)    |
+| Freetrade     | `run start freetrade`  (or `ft`)    |
+| IBKR          | `run start ibkr`                    |
+| Rabobank      | `run start rabobank`                |
+| Schwab        | `run start schwab`                  |
+| Swissquote    | `run start swissquote` (or `sq`)    |
+| Trading 212   | `run start trading212` (or `t212`)  |
+| XTB           | `run start xtb`                     |
+| Investimental | `run start investimental`           |
```

Comment:
Could you move this up, so it is alphabetically sorted?

Reply:
done